clientv3/client.go

```diff
@@ -520,7 +520,12 @@ func toErr(ctx context.Context, err error) error {
 	code := ev.Code()
 	switch code {
 	case codes.DeadlineExceeded:
-		fallthrough
+		if ctx.Err() != nil {
+			err = ctx.Err()
+		} else if strings.Contains(err.Error(), "context deadline exceeded") {
```

Comment:
if the error code is `codes.DeadlineExceeded`, then the error string must contain `context deadline exceeded`, right? why do we need to check it again?

Reply:
also we probably should not convert the server side gRPC error to client side `context.DeadlineExceeded` anyway.